src/color_calibration/__init__.py

```diff
@@ -1 +1,6 @@
-from .match_histograms import match_histograms
\ No newline at end of file
+__version__ = "0.0.1"
```

Comment:
```suggestion
__version__ = "0.0.2"
```
This is updated from @DavidCarlyn's original 0.0.1. Though I might argue to at least go to 0.1.0, since it's changing use.

Reply:
I think `0.1.0` is sensible since this is intended to be used as part of an external workflow, and it's more than just bugfixes.